src/test/java/edu/cornell/kfs/module/purap/fixture/CuB2BShoppingCartFixture.java

```diff
@@ -60,4 +60,14 @@ public CuB2BShoppingCart createB2BShoppingCart() {
 
 		return cart;
 	}
+
+	public CuB2BShoppingCart createB2BShoppingCartWithDuplicateItems() {
```

Comment:
Instead of doing this, I would suggest making a new in this enum and a new item in B2BShoppingCartItemFixture that has duplicate accounts in.  That way you have an enum item that meats your testing needs.  This function may add confusion in the future.